app/models/discount.rb

```diff
@@ -0,0 +1,4 @@
+class Discount < ApplicationRecord
+  validates :tickets_needed, :value, presence: true
+  validates :tickets_needed, :value, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
I would personally split the validations by attributes rather than validation methods, it is easier to determine what validations apply to a specific attribute that way
```suggestion
  validates :tickets_needed, :presence: true, numericality: { only_integer: true, greater_than: 0 }
  validates :value, :presence: true, numericality: { only_integer: true, greater_than: 0 }
```

Reply:
It does look more clear, gonna make another PR and change other models validations, thanks :)